deployment/layoutCmd.py

```diff
@@ -0,0 +1,84 @@
+import os
+from kubernetes import client, config
+import yaml
+
+
+def generate_layout(output_file):
+    # init client
+    config.load_kube_config()
```

Comment:
Can you confirm or ensure there must be a kube_config in the defualt path or there will be a env named "KUBECONFIG"? You should use the parameter to pass the path to the function.

Reply:
Setup the kubectl is the prerequisites.

Reply:
I don't think so. What if user has several kube cluster and of course user has several kubeconfig. He want access the cluster with different kubconfig. At this situation a parameter with kubconfig is needed. And other cmd in paictl all has this parameter. You can simple set a default value as k8s to this parameter. And this won't change the code too much. And I care about this parameter and think it's necessary.

Reply:
Seriously, would admin manage multiple k8s cluster in the same time, on the same host?
Especially in our case, in the same dev-box?
How many times have you ever used the 'KUBECONFIG' parameter?

I agree with what the parameter is for.
But I doubt that we need it.
My point is that: do not confuse user with an option which could ignore.

Reply:
Yes, at least I have used it.

---
In reply to: [252230401](https://github.com/Microsoft/pai/pull/1980#discussion_r252230401) [](ancestors = 252230401)